react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -95,102 +100,66 @@ const CreateMapModal = ({ isOpen, toggle }: CreateMapModalProps) => {
-                )}
-              </FormGroup>
-              {/* TODO_REACT: This part will change once the FileBrowser component is added. https://tacc-main.atlassian.net/browse/WG-208*/}
-              <FormGroup className="row align-items-center">
-                <Label className="col-sm-4">Save Location:</Label>
-                <div className="col-sm-8 text-primary">
-                  {userData ? `/${userData.username}` : 'Loading...'}
+                <div className={`${styles['flexContainer-alt']}`}>
+                  <label htmlFor="save-location-label">Save Location:</label>
+                  <span className="text-primary">/{userData?.username}</span>
                 </div>
-              </FormGroup>
-              <FormGroup className="row mb-2">
-                <Label className="col-sm-4 col-form-label pt-0">
-                  Sync Folder:
-                </Label>
-                <div className="col-sm-8">
-                  <div className="form-check">
-                    <Field
-                      type="checkbox"
-                      name="syncFolder"
-                      id="syncFolder"
-                      className="form-check-input"
-                      checked={values.syncFolder}
-                      onChange={handleChange}
-                    />
-                  </div>
-                  <br />
-                  <br />
-                  <Label
-                    className={`${styles['form-check-label']}`}
-                    for="syncFolder"
-                  >
-                    When enabled, files in this folder are automatically synced
-                    into the map periodically.
-                  </Label>
+                <div className={`${styles['flexContainer']}`}>
+                  <label htmlFor="sync-folder-label">Sync Folder:</label>
+                  <span className={`${styles['check-wrapper']}`}>
+                    <FormikCheck name="syncFolder" label="" description="" />
+                  </span>
+                </div>
+                <div className={`${styles['custom-sync-description']}`}>
+                  When enabled, files in this folder are automatically synced
+                  into the map periodically.
                 </div>
-              </FormGroup>
+              </FieldWrapperFormik>
               {errorMessage && (
                 <div className={`${styles['custom-error-message']}`}>
                   {errorMessage}
                 </div>
               )}
               <ModalFooter className="justify-content-start">
-                <Button size="short" type="secondary" onClick={toggle}>
+                <Button
+                  className="justify-content-center"
+                  size="short"
+                  type="secondary"
+                  onClick={handleToggle}
+                >
                   Close
                 </Button>
                 <Button
+                  className="justify-content-center"
```

Comment:
I see the difference when this (`justify-content-center`) is not added. but I'm perplexed why core-component 'Button" doesn't inherently handle the centering. can we follow up with @wesleyboar on this.

Reply:
https://github.com/TACC-Cloud/hazmapper/blob/master/react/src/core-components/Button/Button.tsx#L78
[3:56](https://tacc-team.slack.com/archives/D064G7JPJ4S/p1708120588929009)
the Button is a c-button (and at least in CEP see https://github.com/TACC/Core-Portal/blob/main/client/src/styles/components/c-button.css#L41-L49), I see a `justify-content: center;` 

but that `justify-content: center;`  is missing in Core-Styles  https://github.com/TACC/Core-Styles/blob/4216132b39358f9b9033be80e931f67a3d134c2e/src/lib/_imports/components/c-button.css
src/lib/_imports/components/c-button.css .    is not there

Reply:
**This is a new bug in Core-Styles.** Consider fixing this globally for your project, with a comment above the CSS to remove the code when it's fixed in Core-Styles (feel free to add the URL of this comment).

Reply:
Actually, anyone is free to fix the bug in Core-Styles. Just open a PR.

Reply:
[The button now has `justify-content: center;`](https://github.com/TACC/Core-Styles/blob/v2.24.0/src/lib/_imports/components/c-button.css#L24), and this is released — [v2.24.0 on GitHub](https://github.com/TACC/Core-Styles/releases/tag/v2.24.0) and [2.24.0 on NPM](https://www.npmjs.com/package/@tacc/core-styles/v/2.24.0). _**Notice**: Tables are redesigned in this release._

Reply:
I see `package.json` loads core-styles with fix. I resolve this thread. If I have done so in error, please unresolve it.